### PR TITLE
added 'user/login' route to API 

### PR DESCRIPTION
**Pull request by madalinazele**
# Justification

#7 

# Implementation

added login functionality

# Testing

used postman with endpoint http://localhost:8080/kitchen/user/login

# Checklist

- [ ] I tested changes to product code in product
- [ ] I considered updates to the README
- [ ] I considered updates to the error code dictionary

### CODE REVIEW

**Review by EnsyFane — APPROVED** [review]
Just make sure to wait until the user/sign-up PR is in and then merge this PR to avoid unnecessary merge conflicts.

**Review by turcudaria — APPROVED** [review]

File: iChef-WebApi/src/main/resources/application.properties
Comment (EnsyFane) [inline iChef-WebApi/src/main/resources/application.properties:2]: @turcudaria @madalinazele @roxanatrufelea @Stefpark77 Can you notify me when you change this config file in a PR? If you change this file I also need to change something on the pipeline. Close this comment when either one of you sees this and tell the others about my request

Comment (Stefpark77) [reply iChef-WebApi/src/main/resources/application.properties:2]: The config file won't be changed by this commit, we will let it be schema,root and 1234 for every push

**Pull request merged**
